src/file.go

```diff
@@ -0,0 +1,62 @@
+package src
+
```

Comment:
Maybe rename this file with a more descriptive name ? like fileHandler.go 
maybe I'm overthinking it but I think it will be better in the long run :)